### DISCUSSION TITLE
✖ Ошибка базы данных: «Доступ запрещен для пользователя« localhost » @« localhost »(с использованием пароля: ДА)» Код ошибки: «1045»

### DISCUSSION THREAD

**Discussion by ivan777000** [givanz/Vvveb#27] [General]
✖ Ошибка базы данных: «Доступ запрещен для пользователя« localhost » @« localhost »(с использованием пароля: ДА)» Код ошибки: «1045» помогите решить данную проблему.

![Безымянный](https://user-images.githubusercontent.com/128806154/227509372-d643de13-8aaa-40f3-883e-85cc84991ae6.png)

**Comment by givanz** [comment]
Hi

The error message says that the configuration is incorrect, from the screenshot it looks like `Database username` is wrong, can you try to enter **root** instead of `localhost` for `Database username` field?

**Reply by ivan777000** [reply]
Я уже пробывал по всякому может вы мне по можете в установки?

**Reply by givanz** [reply]
Do you get the same error message after using `root` for username? 
You can also use sqlite if you have trouble configuring mysql.
